docs/src/quick_start.rst

```diff
@@ -0,0 +1,214 @@
+:term:`instrument plugin <plugin>`.
+
+List of supported instruments
++++++++++++++++++++++++++++++
+
+Numerous plugins are already available for common scientific equipment suppliers, they are referenced in the
+`list of supported instruments <https://github.com/PyMoDAQ/pymodaq_plugin_manager/blob/main/README.md>`_.
+
+.. figure:: /image/quick_start/supported_instruments_list.png
+
+   List of supported instruments.
+
+We can access it from the *Supported instruments* link on the website’s home page.
+
+.. figure:: /image/quick_start/supported_instruments.png
+
+   Link to the list of supported instruments.
+
+Install the software of the supplier
+++++++++++++++++++++++++++++++++++++
+
+To illustrate concretely the procedure, we suppose that we want to control a *Thorlabs Zelux* camera.
+
+.. figure:: /image/quick_start/zelux_camera.png
+   :width: 200
+
+   A Thorlabs Zelux camera.
+
+This camera is controlled with the
+`ThorCam software <https://www.thorlabs.com/software_pages/ViewSoftwarePage.cfm?Code=ThorCam>`_ that is provided by
+Thorlabs. Let’s download and install it.
+
+.. figure:: /image/quick_start/thorcam.png
+
+   The webpage to download Thorcam.
+
+Once it is installed, connect the camera and check that it is working.
+
+.. warning::
+   It is crucial to first check that your instrument can be controlled with the supplier’s software before trying with
+   PyMoDAQ.
+
+The Plugin Manager
+++++++++++++++++++
+
+Once we have checked that our camera is working, we know that the supplier’s drivers, if any, are installed, and that
+the communication between our camera and our computer is working. It is now time to control it with PyMoDAQ.
+
+We have seen that the *Thorlabs* plugin manages this type of camera.
+
+.. figure:: /image/quick_start/supported_instruments_list_thorlabs.png
+
+   The Zelux camera is supported by the Thorlabs plugin.
+
+In this case, we just have to install the Thorlabs plugin in our environment. To do so, we will use the Plugin Manager
+by executing the following command in our terminal
+
+``plugin_manager``
+
+A window is displayed to easily install the plugin.
+
+.. figure:: /image/quick_start/plugin_manager.png
+   :width: 400
+
+   The Plugin Manager interface.
+
+After the plugin installation, we launch a :ref:`DAQ_Viewer_module` with the following command
+
+``daq_viewer``
+
+.. warning::
+   Let’s not forget to disconnect the camera from ThorCam or any other program before trying to communicate with it.
+
+We then have to select the correct *DAQ type* (here 2D because the detector outputs a matrix of pixels) (1), the
+corresponding plugin (2), the specific camera through its serial number (3), initialize the communication with the
+camera (4), and get a snapshot (5).
+
+.. figure:: /image/quick_start/quick_start_image_thorlabs.png
+   :width: 800
+
+   Snapshot from a Thorlabs camera equipped with an objective, in a DAQ Viewer.
+
```

Comment:
I like your snapshot!

Reply:
It took me a bit of time to know what I would snapshot! :P